homeassistant/components/thermostat/homematic.py

```diff
@@ -108,21 +144,48 @@ def device_state_attributes(self):
                 "battery": self._battery,
                 "mode": self._mode}
 
+    @property
+    def min_temp(self):
```

Comment:
Alrighty, sounds good.

On Sun, Apr 10, 2016 at 2:52 PM, Markus Peter <notifications@github.com>
wrote:

> In homeassistant/components/thermostat/homematic.py
> <https://github.com/balloob/home-assistant/pull/1783#discussion_r59142246>
> :
>
> > @@ -108,21 +144,48 @@ def device_state_attributes(self):
> >                  "battery": self._battery,
> >                  "mode": self._mode}
> >
> > +    @property
> > +    def min_temp(self):
>
> It can also be set to a temperature (that open/closed functionality is not
> terribly useful in my opinion).
> This is true for all eq3 products.
> They also all have an automatic mode with a builtin time program as well
> as a manual mode where that functionality is disabled, and can be set to a
> target temperature in both auto and manual mode (which is the functionality
> that is used by this component).
> The mode flag is queried by the component and could potentially be used in
> automation rules (probably with the time program on the thermostat itself
> deleted).
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/1783/files/a4500a225593995569e075456dd7a68777bced66#r59142246>
>



-- 
PaulusSchoutsen.nl
It's nice to be important but it's more important to be nice.